### ISSUE TITLE
Slot for footer ? Event for show/hide ?

### ISSUE THREAD

**Issue by scorphious** [nathanreyes/v-calendar#316]
Is it possible to include a slot for datepicker footer ? Right below the day options.
Personally I added a "Go to today" button there, bottom right.

Also - in 0.9.7 there are events called "will-appear" and "did-appear", same for disappear. But I see no such thing in the dev versions. Are they going to be implemented?

**Comment by ianbunn** [comment]
Hi @scorphious I noticed from your comment that you were able to add a "Go to today" button in the footer. Would you mind sharing how you accomplished that?

Thank you for your time!

**Comment by scorphious** [comment]
Hey @ianbunn, I dont have a code example as we moved on from this component and made a datepicker ourselves. But I can tell you how I did it:
After the v-calendar datepicker had rendered (usually in the $nextTick), we created as simple button element like so - document.createElement("button"), with all it needs (innerText, onClick handler, class). Then targeted the dropdown in question and internally its div with the class "c-pane", which is the main contents of it.

And then just append the created button to that c-pane HTML element, nothing else.

The onClick handler can be customized to your needs. For us it first moved the calendar to the date in question and, if the date wasnt disabled in any way, selected it programmatically.

Since v-calendar didnt offer any support for any of those mentioned functionalities, then we sort of.. hacked away :)) a glimpse of how the "move to today date in the calendar" was built: this.$refs.datepicker.$children[0].$children[0].$children[0].$children[0].move({ day: date.getDate(), month: date.getMonth() + 1, year: date.getFullYear() });

That was in 0.9.7 I believe, maybe things have changed since then.
